sphinx/builders/linkcheck.py

```diff
@@ -127,7 +128,7 @@ def init(self) -> None:
 
         # create queues and worker threads
         self.rate_limits = {}  # type: Dict[str, RateLimit]
-        self.wqueue = queue.PriorityQueue()  # type: queue.PriorityQueue
+        self.wqueue = queue.PriorityQueue()  # type: queue.PriorityQueue[CheckRequest]
```

Comment:
This changes the type of `wqueue`. But I have a plan to make this attribute a dummy (refs: https://github.com/sphinx-doc/sphinx/pull/8835/files#r571341453). So this does not cause a crash.

Reply:
I determined to keep compatibility with old versions. So old tuple is still accepted.